.github/workflows/tests.yml

```diff
@@ -14,16 +14,41 @@ jobs:
       - uses: actions/checkout@v1
       - name: Select Xcode 11.2
         run: sudo xcode-select -s /Applications/Xcode_11.2.app
-      - name: Install buildifier
-        run: brew install buildifier
       - name: Build and Test
         run: bazelisk test --local_test_jobs=2 //...
       - uses: actions/upload-artifact@v1
         if: failure()
         with:
           name: bazel-testlogs
           path: bazel-testlogs
+  check_docs:
+    name: Check Starlark
+    runs-on: macOS-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Select Xcode 11.2
```

Comment:
small nit: is it necessary to select xcode versions before running buildifier?

Reply:
No clue